src/ElementProxy.js

```diff
@@ -12,26 +12,31 @@ import {NodeProxy} from './types'
 export class ElementProxy {
   _node: HTMLElement;
 
+  // Expose these so they can be stubbed for unit test
+  _emitMount: Function;
+  _emitUnmount: Function;
+
   constructor (node: HTMLElement) {
     this._node = node
   }
 
   emitMount (fn: Function): void {
-    emitMount(this._node, fn)
+    this._emitMount(this._node, fn)
   }
 
   emitUnmount (fn: Function): void {
-    emitUnmount(this._node, fn)
+    // Rely on prototype method so it can be stubbed for unit test
+    this._emitUnmount(this._node, fn)
```

Comment:
After a little more thought, I think it makes sense to test mount events with `ElementProxy` since `ElementProxy` is responsible for both adding the listeners and emitting the events, in contrast to `mountable` which is just responsible for emitting. My plan is to back out `_emitMount` and `_emitUnmount` and declare those tests to be added in a follow-up PR.

Sound OK?